### PR TITLE
Fixed waits for the Show rows field

### PR DESCRIPTION
**Pull request by denysord88**

### CODE REVIEW

**Comment by denysord88** [comment]
@agozhiy @Atri0 please review the PR.

**Comment by Atri0** [comment]
+1 LGTM

File: src/test/java/org/apache/drill_web_test_framework/web_ui/pages/QueryResultsPage.java [hunk 1]
```diff
@@ -145,6 +145,19 @@ public String getQueryProfile() {
   public QueryResultsPage showResultRows(String rowsCount) {
     rowsPerPage.click();
     rowsPerPage.findElement(By.xpath("//*[text() = '" + rowsCount + "']")).click();
+    if(!rowsCount.equals("ALL")) {
+      waitForRowsCount((Integer.parseInt(rowsCount) + 1));
+    } else {
+      String allRowsCount = pageRowsInfo.getText()
+          .substring(pageRowsInfo.getText().indexOf("of "))
+          .replaceAll("[a-z ]", "");
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/web_ui/pages/QueryResultsPage.java]: Maybe it's better to use regex fully?

```suggestion
    Matcher matcher = Pattern.compile("([\\d]+)\\w*entries")
        .matcher(pageRowsInfo.getText());
    if (matcher.find()) {
      allRowsCount = matcher.group(1);
    }
```

File: src/test/java/org/apache/drill_web_test_framework/web_ui/pages/QueryResultsPage.java [hunk 2]
```diff
@@ -145,6 +145,19 @@ public String getQueryProfile() {
   public QueryResultsPage showResultRows(String rowsCount) {
     rowsPerPage.click();
     rowsPerPage.findElement(By.xpath("//*[text() = '" + rowsCount + "']")).click();
+    if(!rowsCount.equals("ALL")) {
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/web_ui/pages/QueryResultsPage.java]:
```suggestion
    if (!rowsCount.equals("ALL")) {
```

File: src/test/java/org/apache/drill_web_test_framework/web_ui/steps/QueryResultsSteps.java [hunk 3]
```diff
@@ -109,4 +109,9 @@ public QueryResultsSteps showResultRows(String rowsCount) {
     BasePage.getPage(QueryResultsPage.class).showResultRows(rowsCount);
     return getSteps(QueryResultsSteps.class);
   }
+
+  public QueryResultsSteps waitForRowsCount(int rowsCount) {
+    BasePage.getPage(QueryResultsPage.class).waitForRowsCount(rowsCount);
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/web_ui/steps/QueryResultsSteps.java]:
```suggestion
    BasePage.getPage(QueryResultsPage.class)
        .waitForRowsCount(rowsCount);
```

File: src/test/java/org/apache/drill_web_test_framework/web_ui/tests/query/ResultsTableTest.java [hunk 4]
```diff
@@ -143,7 +143,8 @@ public void testRowsOutputCounter() {
 
   @Test(groups = {"functional"})
   public void testShowEntries() {
-    assertEquals(querySteps.runSQL("select * from cp.`employee.json` LIMIT 150").getTable().size(), 11);
+    querySteps.runSQL("select * from cp.`employee.json` LIMIT 150");
+    queryResultsSteps.waitForRowsCount(11);
```
Comment (agozhiy) [inline src/test/java/org/apache/drill_web_test_framework/web_ui/tests/query/ResultsTableTest.java]:
```suggestion
    querySteps.runSQL("select * from cp.`employee.json` LIMIT 150")
       .waitForRowsCount(11);
```

**Review by agozhiy — CHANGES_REQUESTED** [review]

**Comment by agozhiy** [comment]
Just a few minor comments )

File: src/test/java/org/apache/drill_web_test_framework/web_ui/pages/QueryResultsPage.java (hunk 2 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/web_ui/pages/QueryResultsPage.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/web_ui/pages/QueryResultsPage.java (hunk 1 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/web_ui/pages/QueryResultsPage.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/web_ui/steps/QueryResultsSteps.java (hunk 3 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/web_ui/steps/QueryResultsSteps.java]: Done

File: src/test/java/org/apache/drill_web_test_framework/web_ui/tests/query/ResultsTableTest.java (hunk 4 above)
Comment (denysord88) [reply src/test/java/org/apache/drill_web_test_framework/web_ui/tests/query/ResultsTableTest.java]: Done

**Comment by denysord88** [comment]
@agozhiy please review changes.

**Review by agozhiy — APPROVED** [review]

**Comment by agozhiy** [comment]
+1. Thanks for the changes, you can squash the commits and merge the PR now.

**Pull request merged**